bot.js

```diff
@@ -682,6 +683,75 @@ function command(message) {
         }
         message.reply(help);
         log.notice("Issued help message.");
+    } else if (message.content.startsWith(config.commands.help + " ")) {
+        // Per-command helptext.
+        log.notice("Received help-for-command command.");
+        const target = message.content.substring(
+            config.commands.help.length + 1
+        );
+        log.info("Help is for command: " + target);
+        if (Object.keys(config.commandDetails).includes(target)) {
+            const detailsObject = config.commandDetails[target];
+            let response;
+
+            // Assemble a title using the configured 'trigger phrase' for the command
+            if (detailsObject.internalKey == null) {
+                // If we haven't been told how to, log an error and use the passed key instead.
+                log.error(
+                    "Detailed helptext spec for command " +
+                        target +
+                        " does not include an internalKey!"
+                );
+                response = "> **The " + target + " command**\n";
+            } else {
+                response =
+                    "> **" +
+                    config.commands[detailsObject.internalKey] +
+                    "**\n";
+            }
+
+            // Now, the subtitle.
+            if (detailsObject.subtitle == null) {
+                // If there is no subtitle, log a warning.
+                log.warning(
+                    "Detailed helptext spec for command " +
+                        target +
+                        " does not include a subtitle!"
+                );
+                response += "> \n";
+            } else {
+                response += "> " + detailsObject.subtitle + "\n> \n";
+            }
+
+            // And finally, the body.
+            if (detailsObject.lines == null) {
+                // If there is no body, log an error.
+                log.error(
+                    "Detailed helptext spec for command " +
+                        target +
+                        " does not include a lines array!"
+                );
+                response += "> I don't really have anything to say here.";
+            } else {
+                response += detailsObject.lines.map(x => "> " + x).join("\n");
+            }
+
+            // Now send the response.
+            sendLongMessage(message.channel, response);
```

Comment:
Is sendLongReply something that could have been used here (but sendLongMessage was chosen instead)?

Reply:
sendLongReply could have been used as well here without issues, yes (`sendLongReply(message, response)`)

I chose sendLongMessage instead because I disliked how @za419, <title> looked... I suppose I could have added a new line at the beginning instead.

But overall I think sendLongMessage is appropriate here because helptext isn't necessarily person-specific - There's no reason it has to be directed at the sending user.

Of course, then you'd ask, why does short helptext use reply?
... And the answer is it was one of the first commands I wrote, and I didn't know how to send a message in the same channel without replying, and never bothered fixing it. I suppose they should be consistent though...